bundle/bundle.go

```diff
@@ -96,6 +96,15 @@ func (i *BaseImage) DeepCopy() *BaseImage {
 	return &i2
 }
 
+// DigestedRef returns an image reference which can be used to pull by digest using Docker or Kubernetes.
+// If no digest is available, only the image will be returned and the boolean value will be false.
+func (i *BaseImage) DigestedRef() (string, bool) {
+	if i.Digest == "" {
+		return i.Image, false
```

Comment:
I think returning false here is not accurate, as `i.Image` can be a digested reference itself.  We should test if the image name is a digested reference here and return the result.

Reply:
I'm on the fence about this. The spec states that

> If the `contentDigest` is not present, the runtime SHOULD report an error so the user is aware that there is no `contentDigest` provided.

If an image contains what we interpret to be a digest, then some runtimes could certainly opt to use this in lieu of an explicit `contentDigest`. But having the boolean correspond to whether `contentDigest` has actually been set seems most in line with the spec as written.